app/views/categories/show.html.erb

```diff
@@ -1,19 +1,19 @@
 <div class="menu">
-<a href="/categories" class="text-light"><svg xmlns="http://www.w3.org/2000/svg" width="26" height="36" fill="currentColor" class="bi bi-arrow-left" viewBox="0 0 16 16">
-  <path fill-rule="evenodd" d="M15 8a.5.5 0 0 0-.5-.5H2.707l3.147-3.146a.5.5 0 1 0-.708-.708l-4 4a.5.5 0 0 0 0 .708l4 4a.5.5 0 0 0 .708-.708L2.707 8.5H14.5A.5.5 0 0 0 15 8z"/>
-</svg></a>
+  <a href="/categories" class="text-light"><svg xmlns="http://www.w3.org/2000/svg" width="26" height="36" fill="currentColor" class="bi bi-arrow-left" viewBox="0 0 16 16">
+      <path fill-rule="evenodd" d="M15 8a.5.5 0 0 0-.5-.5H2.707l3.147-3.146a.5.5 0 1 0-.708-.708l-4 4a.5.5 0 0 0 0 .708l4 4a.5.5 0 0 0 .708-.708L2.707 8.5H14.5A.5.5 0 0 0 15 8z"/>
+    </svg>
+  </a>
 <h2 class="text-light text-center">Details</h2>
 <span></span> 
 </div>
 <div class="col-10 offset-1 mt-5 p-2"> 
   <% if notice %>
-  <p class="alert alert-success"><%= notice %></p>
+    <p class="alert alert-success"><%= notice %></p>
   <% end %>
   <% if alert %>
-  <p class="alert alert-danger"><%= alert %></p>
+    <p class="alert alert-danger"><%= alert %></p>
   <% end %>
-    <div class="row row-cols-1 shadow p-3 mb-5 bg-body opacity-75"> 
-        
+    <div class="row row-cols-1 shadow p-3 mb-5 bg-body opacity-75">  
       <div class="border-bottom mb-3"> 
         <h5><img src="<%= @category.icon%>" ><span><%= @category.name %></span></h5>  
       </div> 
```

Comment:
### Your Transactions Page
![image](https://user-images.githubusercontent.com/93039359/232199216-ac706192-fe2b-4ad6-b272-d16df5c0a3b8.png)

### Design Transactions Page
![image](https://user-images.githubusercontent.com/93039359/232199234-6e65bc63-a01d-4c1e-8a59-c708db63248f.png)

- [ ] It would be nice if you follow the design and make your transactions page looks like the given design to make your project look professional :+1:

Reply:
this is the details page of the selected food you can go to the transaction page by clicking the link I add to this page

Reply:
Hey @AhmadiGu I hope you are doing well :blush:

You can ask your partner who can make ac account for you from their country :+1: